### PR TITLE
Add technical details

### PR DESCRIPTION
**Pull request by fabaff**
Add some details from https://developer.sony.com/develop/spresense/docs/introduction_en.html#_main_board

### CODE REVIEW

**Comment by dhalbert** [comment]
@kamtom480 Could you review these changes?

**Review by kamtom480 — CHANGES_REQUESTED** [review]

File: _board/spresense.md [hunk 1]
```diff
@@ -13,14 +13,35 @@ features:
...
+* Pins and LEDs
+  * Multiple GPIO (UART, SPI, I2C, I2S)
+  * 2 ADC channels
+  * 4 Application LED (Green)
+  * Power LED (Blue)
+* Micro USB serial port
+
+## Misc
+
+* [Pinout](https://developer.sony.com/develop/spresense/docs/introduction_en.html#_main_board)
```
[the remark this answers is not in the document]
Comment (fabaff) [reply _board/spresense.md]: Added, thanks.

File: _board/spresense.md [hunk 2]
```diff
@@ -13,14 +13,35 @@ features:
...
+* PCB with small footprint
+* Dedicated camera connector
+* GNSS (GPS) antenna
+* Pins and LEDs
+  * Multiple GPIO (UART, SPI, I2C, I2S)
+  * 2 ADC channels
+  * 4 Application LED (Green)
+  * Power LED (Blue)
+* Micro USB serial port
+
```
[the remark this answers is not in the document]
Comment (fabaff) [reply _board/spresense.md]: The core features of the Spresense extension board are now mentioned.

File: _board/spresense.md [hunk 3]
```diff
@@ -13,14 +13,38 @@ features:
...
+## Technical details
+
+* CXD5602 Processor (Arm Cortex-M4F 156 MHz)
+* 8 MB Flash memory, SRAM 1.5 MB
+* PCB with small footprint
+* Dedicated camera connector
+* GNSS (GPS) antenna
+* Pins and LEDs
+  * Multiple GPIO (UART, SPI, I2C, I2S)
+  * 2 ADC channels
```
Comment (kamtom480) [inline _board/spresense.md:36]: There are 2 ADC channels on the main board, but 6 channels is on the extension board. Maybe it is worth adding this information here.

**Review by kamtom480 — CHANGES_REQUESTED** [review]
Thank you for your previous changes. Here I have two more suggestions.

File: _board/spresense.md (hunk 3 above)
Comment (fabaff) [reply _board/spresense.md]: Thanks

File: _board/spresense.md [hunk 4]
```diff
@@ -13,14 +13,38 @@ features:
...
+## Technical details
+
+* CXD5602 Processor (Arm Cortex-M4F 156 MHz)
+* 8 MB Flash memory, SRAM 1.5 MB
+* PCB with small footprint
+* Dedicated camera connector
+* GNSS (GPS) antenna
+* Pins and LEDs
+  * Multiple GPIO (UART, SPI, I2C, I2S)
+  * 2 ADC channels (6 additional ADC channels are located on the extension board)
```
Comment (kamtom480) [inline _board/spresense.md:36]: Spresense has 6 ADC channels. On the main board, 2 (A2 and A3) are available on the pin socket, and the other 4 via the B2B connector. On the extension board, all 6 (A0 - A5) ADC channels are available on the pin socket.

In order not to complicate this, you can write:
```suggestion
  * 6 ADC channels
```
What do you think?

**Review by makermelissa — APPROVED** [review]

**Pull request merged**